.github/CODEOWNERS

```diff
@@ -13,16 +13,7 @@
 #
 
 # Global owners, will be the owners for everything in the repo. Membership is tracked via https://github.com/open-telemetry/community/blob/main/community-members.md 
-*   @open-telemetry/specs-approvers
-
-# Trace owners (global + trace)
-opentelemetry/proto/trace/    @open-telemetry/specs-approvers @open-telemetry/specs-trace-approvers
-
-# Metrics owners (global + metrics)
-opentelemetry/proto/metrics/  @open-telemetry/specs-approvers @open-telemetry/specs-metrics-approvers
-
-# Logs owners (global + logs)
-opentelemetry/proto/logs/     @open-telemetry/specs-approvers @open-telemetry/specs-logs-approvers
+*   @open-telemetry/spec-sponsors
 
 # Profiles owners (global + profiles)
-opentelemetry/proto/profiles  @open-telemetry/specs-approvers @open-telemetry/profiling-maintainers
+opentelemetry/proto/profiles  @open-telemetry/spec-sponsors @open-telemetry/profiling-maintainers
```

Comment:
You need to give sponsors write permissions I think

Reply:
Updated the PR description to indicate that upon merging I need to followup with giving write permissions.